src/sentry/snuba/sessions_v2.py

```diff
@@ -122,6 +123,12 @@ def extract_from_row(self, row, group):
         return 0
 
 
+def finite_or_none(val):
+    if not math.isfinite(val):
+        return None
+    return val
```

Comment:
We define some custom json encodings for python types in sentry.utils.json. This could be added to `better_default_encoder` and be available in all of our endpoints.